src/common/utils/index.ts

```diff
@@ -54,10 +54,12 @@ export const formatDecimal = (num: number, places = 2): string => {
     return '-';
   }
 
-  return num.toLocaleString(undefined, {
-    minimumFractionDigits: places,
-    maximumFractionDigits: places,
-  });
+  return num
+    .toLocaleString(undefined, {
+      minimumFractionDigits: places,
+      maximumFractionDigits: places,
+    })
+    .replace('-0', '0'); // floating point precision errors can cause expected-zero values to return "-0"
```

Comment:
Hrm, you could try adding `signDisplay: 'negative'` to the options above instead. Not 100% sure if that'll work.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat
![image](https://user-images.githubusercontent.com/206364/162841819-3a24e94b-19a7-4c59-9b1f-58d6c46b6975.png)

Reply:
I'm facing an error when trying to use `signDisplay`--it looks like relating to https://github.com/microsoft/TypeScript/issues/40038 --where `signDisplay` is missing from the NumberFormatOptions (in the file below). The thread links to a fix, so I'm not 100% sure why it's failing here :/

For now, I might add a `TODO` with a link to this issue, unless there's an easy way to resolve this issue. 


`/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/node_modules/typescript/lib/lib.es5.d.ts`

Reply:
Ah, interesting.  It's possible that `.toLocaleString()` doesn't take the same arguments as `Intl.NumberFormat` though I found a doc that suggested it does.  But you could try:

```
const formatter = Intl.NumberFormatter(undefined, {
      minimumFractionDigits: places,
      maximumFractionDigits: places,
      signDisplay: 'negative'
});

return formatter.format(num);
```

Not a big deal though, so if it's not trivial to figure out feel free to ignore.

Reply:
Oh, Actually I just noticed the little flask icon next to the "negative" docs apparently means that the functionality is experimental.  Probably best to avoid relying on it.  Feel free to just keep your existing code.

Reply:
Ah true, there's the `exceptZero` option which seems like it should also do what we want. 

Unfortunately, I get the same error when trying the `const formatter = Intl.NumberFormat(undefined, {...` approach. Interestingly though, clicking through on each key to see what interface they come from leads me to different source files. 

`signDisplay` brings me to: 
`/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/node_modules/typescript/lib/lib.es2020.intl.d.ts`

while

`minimumFractionDigits` and `maximumFractionDigits` bring me to: 
`/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/node_modules/typescript/lib/lib.es5.d.ts`. 

I wonder if maybe some packages are conflicting with each other? I'm not too sure what I'm looking at :/

Reply:
Yeah, I was thinking `exceptZero` would work at first, but I think it might add a `+` to positive numbers, which we don't want. 😁 

I think those files es2020 vs es5 files probably mean that `signDisplay` was added in [ECMAScript 2020](https://en.wikipedia.org/wiki/ECMAScript#11th_Edition_%E2%80%93_ECMAScript_2020) (the JavaScript standard released in 2020) vs. the others which have been around since [ECMAScript 5](https://en.wikipedia.org/wiki/ECMAScript#5th_Edition) (the JavaScript standard released in 2009).

I think maybe we are only [letting TypeScript build with ES2019 libraries](https://github.com/covid-projections/covid-projections/blob/00994c38a508fe8b496608787de11e17a57bd883/tsconfig.json#L5).  If we changed that to 2020, then maybe the error would go away?  Not sure.  I'm also not sure if we intentionally chose 2019 intentionally.  I'd want to check with Pablo before changing it.  But feel free to leave it as-is.

Reply:
Oh, yeah we wouldn't want that 😅 

I see, that makes sense. Thanks so much for this explanation 🙌 I'll leave as-is with a TODO for now